core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -144,7 +144,7 @@ case class LogReadResult(info: FetchDataInfo,
   def withEmptyFetchInfo: LogReadResult =
```

Comment:
This is not used as far as I can see, do we still need it?

Reply:
You're right, this is unused -> removed